### ISSUE TITLE
Form submissions not working

### ISSUE THREAD

**Issue by TylerMRoderick** [danurbanowicz/eleventy-netlify-boilerplate#63]
**Describe the bug**
I deployed this project to Netlify, but the form submissions were not working for me. The "forms" tab in my Netlify dashboard also didn't show any active forms.

**Solution**
On the `<form>` tag, I replaced `netlify` with `data-netlify="true"` and that fixed the problem. The Netlify forms documentation still seems to support the `netlify` attribute alone, so I'm not sure why this was an issue. Either way, I'm just leaving a note here in case anyone else runs into the same issue.

Thanks for the boilerplate 👍

**Comment by danurbanowicz** [comment]
Yeah this is weird; just tested with a fresh deploy of this repo and the resulting form is not working (it returns a 404 on submit), however I got the same error when using the alternative `data-netlify="true"` format also. So it might not be related to that.

In both cases, the form is recognised by Netlify during the build (the form shows in the build log) but the resulting form does not appear to be handling the submissions.

**Comment by danurbanowicz** [comment]
This seems to be an issue with Netlify's form detection not recognising the contact form on a newly created site.

Interestingly, disabling and immediately re-enabling form detection in: `Build & Deploy > Post processing > Form detection` caused the contact form to be recognised and to accept submissions.

**Comment by danurbanowicz** [comment]
I'm going to close this now as [Netlify have been notified](https://community.netlify.com/t/netlify-forms-new-forms-not-being-recognised/32825) about the problem and we need to wait on them. I'll post any update here.

If anyone else is having problems with the contact form being recognised on new Netlify deployments, please reply to this or open a new issue.

**Comment by danurbanowicz** [comment]
Netlify support have confirmed of a problem at their end causing Netlify forms to not be recognised during builds, and to fail silently.

https://answers.netlify.com/t/netlify-forms-new-forms-not-being-recognised/32825/17?u=danurbanowicz

Removing the `name` attribute from the form submit button causes the form to be recognised. We're still not sure why this is.

**Comment by danurbanowicz** [comment]
This has been temporarily fixed by 7de2dcd
